### PR TITLE
Added touch commands

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

**Comment by imurchie** [comment]
There is a merge conflict artifact in there.

File: commands-yml/commands/session/back.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+name: Go Back
+short_description: Navigate backwards in the browser history, if possible (Web context only)
+
+example_usage:
+  java:
+    |
+      driver.back();
```
Comment (SrinivasanTarget) [inline commands-yml/commands/session/back.yml]: @dpgraham AFAIK, i'm not aware of back method in iOS or AndroidDriver in java client. Would review the entire PR once before we get this in. So hold on for some moment.

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

File: commands-yml/commands/session/back.yml (hunk 1 above)
Comment (dpgraham) [reply commands-yml/commands/session/back.yml]: @SrinivasanTarget any update on this?

File: commands-yml/commands/interactions/touch/double-tap.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO Dotnet documentation link
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: true
+    uiautomation: true
+  android:
+    uiautomator2: true
+    uiautomator: true
```
Comment (SrinivasanTarget) [inline commands-yml/commands/interactions/touch/double-tap.yml]: Isn't it false? we use TouchAction API for doubletap not TouchActions.

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

File: commands-yml/commands/interactions/touch/double-tap.yml (hunk 2 above)
Comment (dpgraham) [reply commands-yml/commands/interactions/touch/double-tap.yml]: So it's not supported in Android?

Comment (SrinivasanTarget) [reply commands-yml/commands/interactions/touch/double-tap.yml]: yes. `doubleTap` is supported via TouchAction.

File: commands-yml/commands/session/back.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO Dotnet documentation link
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: true
+    uiautomation: true
+  android:
+    uiautomator2: true
+    uiautomator: true
```
Comment (SrinivasanTarget) [inline commands-yml/commands/session/back.yml]: Should be false.

File: commands-yml/commands/interactions/touch/double-tap.yml (hunk 2 above)
Comment (dpgraham) [reply commands-yml/commands/interactions/touch/double-tap.yml]: Should it be:

```
 android:
    uiautomator2: true
    uiautomator: true
```

File: commands-yml/commands/session/back.yml (hunk 3 above)
Comment (dpgraham) [reply commands-yml/commands/session/back.yml]: So:

```
 android:
    uiautomator2: true
    uiautomator: false
```

File: commands-yml/commands/interactions/touch/double-tap.yml (hunk 2 above)
Comment (SrinivasanTarget) [reply commands-yml/commands/interactions/touch/double-tap.yml]: Should be:
```
ios:
   xcuitest: false
   uiautomation: false
android:
  uiautomator2: false
   uiautomator: false
```

Comment (SrinivasanTarget) [reply commands-yml/commands/interactions/touch/double-tap.yml]: It should be like above for this whole PR everywhere

**Pull request merged**